API/Helpers/PagedList.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace Helpers
+{
+    public class PagedList<T> : List<T>
+    {
+        public PagedList(
+            IEnumerable<T> items,
+            int count,
+            int pageNumber,
+            int pageSize
+        )
+        {
+            CurrentPage = pageNumber;
+            TotalPages = (int) Math.Ceiling(count / (double) pageSize);
```

Comment:
TotalPages is more like a client side concept. Unless you use Asp.Net MVC, which does need this to render the page, otherwise can be removed and let the client side to decide. But nothing is wrong here if you keep it.